rules/c2/tool_transfer/shell.yara

```diff
@@ -184,6 +186,10 @@ rule obsessive_dropper: critical {
     $cmd_echo      = "echo" fullword
     $cmd_chmod     = "chmod" fullword
 
+    $not_deckar01_bootstrap1 = "gem install bundler"
```

Comment:
in addition, mark this rule `high`, as clearly there are false positives here.

Reply:
Updated in [`4260efd` (#694)](https://github.com/chainguard-dev/malcontent/pull/694/commits/4260efda23707a4e7d4a8539902b41a090f16684).